catalog/data_catalogs.go

```diff
@@ -4,14 +4,16 @@ import (
 	"context"
 
 	"github.com/databricks/databricks-sdk-go"
+	"github.com/databricks/databricks-sdk-go/service/catalog"
 	"github.com/databricks/terraform-provider-databricks/common"
 )
 
 func DataSourceCatalogs() common.Resource {
 	return common.WorkspaceData(func(ctx context.Context, data *struct {
 		Ids []string `json:"ids,omitempty" tf:"computed,slice_set"`
 	}, w *databricks.WorkspaceClient) error {
-		catalogs, err := w.Catalogs.ListAll(ctx)
+		var req catalog.ListCatalogsRequest = catalog.ListCatalogsRequest{}
```

Comment:
why we need to have variable here, why not simple `ListAll(ctx, catalog.ListCatalogsRequest{})` ?